lib/services/user.service.js

```diff
@@ -0,0 +1,87 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export default class UserService {
+    constructor() {
+        this.users = [{
+            id: uuidv4(),
+            login: 'c lonely user',
+            password: 'hide me',
```

Comment:
Sensitive data should not be present in response (password)
Also internal data should not be present (isDeleted)

Reply:
Done